app/pods/agenda/agendaitems/agendaitem/index/template.hbs

```diff
@@ -46,8 +46,8 @@
 {{#if this.model.showAsRemark}}
   <Mandatees::MandateesDomainsPanel
     @mandatees={{this.mandatees}}
+    @submitter={{this.submitter}}
     @allowEditing={{false}}
-    @onSave={{this.saveMandatees}}
```

Comment:
De onSave hebben we wel nodig zou ik denken ?

Reply:
Ik had het nagekeken: if `allowEditing={{false}}` wordt niet gesaved.
Ik heb de regel toch maar teruggezet: het lijkt me te klein om opnieuw te verifieren.

Reply:
Je had gelijk hoor, die flag verbergt de dit knop, dus kan je ook niet saven